include/mapnik/filesystem.hpp

```diff
@@ -23,7 +23,7 @@
 #ifndef MAPNIK_FILESYSTEM_HPP
 #define MAPNIK_FILESYSTEM_HPP
 
-#if defined(__cpp_lib_filesystem) && !defined(USE_BOOST_FILESYSTEM)
+#if (__cplusplus >= 201703L) && !defined(USE_BOOST_FILESYSTEM)
```

Comment:
This should allow compiling with MSVC without specifying the `/Zc:__cplusplus` option:
```suggestion
#if ((__cplusplus >= 201703L) || (defined(_MSVC_LANG) && _MSVC_LANG >= 201703L)) && !defined(USE_BOOST_FILESYSTEM)
```

Reply:
Let me check it tomorrow. Normally this shouldnt be needed.

Reply:
No problem, just trying to be thorough here, I mentioned it in case using `/Zc:__cplusplus` with `MSVC` was not something you wanted to do.